docs/ss_slide_001.md

```diff
@@ -7,4 +7,4 @@
 ## * Qualitative differences in behavior also become apparent at different step sizes for some patterns. 
 
 {:style="text-align:center;"}
-[Previous slide](https://rivesunder.github.io/yuca/ss_slide_000.md) -- [Next slide](https://rivesunder.github.io/yuca/ss_slide_002)
+[Previous slide](https://rivesunder.github.io/yuca_docs/ss_slide_000.md) -- [Next slide](https://rivesunder.github.io/yuca_docs/ss_slide_002)
```

Comment:
Let's go ahead and revert the links to the gh-pages to point to `https://rivesunder.github.io/yuca` instead of yuca_docs. This should decrease navigation confusion while still offloading the larger assets storage to yuca_docs.